packaging/systemd/microshift-containerized-configure-selinux

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Is this file still needed / up-to-date?

If we're running containerized, only the crio.sock should be bind-mounted from the host (plus the `/var/lib/microshift` datadir if we're using hostdir instead of voliumes). So why do we still need to create these other dirs and adjust SELinux policies?

Reply:
We don't need to adjust the selinux anymore. From @mangelajo comment. I will remove the files as well. It was based on the assumption that the users didn't download the rpms and would be only using the `microshift-containerized.service`
_"Let's assume that people will download the rpms for microshift via dnf, which will include both microshift-containerized + the selinux modules compiled."_

Reply:
Wait... I think we're confusing deployment models here:
The containerised deployment is for users who want to deploy MIcroShift at runtime on top of a R4E container host. That is, they cannot dynamically install an .RPM (that would need to be baked into the rpm-ostree).

Reply:
Yes, they can't dynamically install it, but if you want to bake it, then we need it also to be in the RPM?.

In any case I don't think we should be running make for the selinux files on installation, or downloading seliux-policy-devel etc... in that case we should probably provide the selinux compiled policy, if that's standard across kernel versions, which I don't know.

May be the container itself should be built by installing the RPM we have?, then the user just downloads the .service file? Can we install an selinux policy from inside a privileged container?

Reply:
According to this, can’t just people install an rpm? We made a subpackage for the microshift container service which will also pull the selinux defs: https://developers.redhat.com/blog/2020/03/12/how-to-customize-fedora-coreos-for-dedicated-workloads-with-ostree

Reply:
What I meant is we have two deployment models:
1) The .rpm-based installation, which is for embedding into rpm-ostrees and runs MicroShift as binary on the host / non-containerized. We have corresponding SELinux rules for that.
2) The containerized deployment, which is for users who _don't_ want to embed into an rpm-ostree but instead use a container host and add/remove MicroShift at runtime - without reboot. This is where it doesn't make sense to install the sytemd unit via an .rpm. Instead, it'll need to be created via configuration management resp. from the application management agent.